IDEAS/Examples/Tutorial/DetailedHouse/package.mo

```diff
@@ -0,0 +1,107 @@
+within IDEAS.Examples.Tutorial;
+package DetailedHouse "Package with example for how to built a simple house, using the detailed building envelope component
+  models, HVAC and other components"
+  extends Modelica.Icons.ExamplesPackage;
+
+  annotation (Documentation(info="<html>
+<p>
+This package contains examples with step-by-step instructions for how to build a system model
+for a simple house with detailed building envelope component models, HVAC and other components.
+It serves as a demonstration case of how the <code>IDEAS</code> library can be used.
+</p>
+<p>
+The goal of this exercise is to become familiar with Modelica and the IDEAS library.
+Since the IDEAS library components are typically used by combining several components graphically,
+the use of equations falls outside of the scope of this exercise.
+</p>
+<p>
+For this exercise you will create a model of a simple house,
+consisting of a heating system, detailed building envelope component models, and a ventilation model.
+The exercise starts from a template file that should not produce any errors.
+This file will be extended in several steps, adding complexity.
+In between each step the user should be able to simulate the model,
+i.e., no errors should be produced and simulation results may be compared.
+</p>
+<p>
+The model has been created in the following stages:
+</p>
+<ol>
+<li>
+<a href=\"modelica://IDEAS.Examples.Tutorial.DetailedHouse.DetailedHouse1\">
+IDEAS.Examples.Tutorial.DetailedHouse.DetailedHouse1</a>
+represents a basic building model that includes a single zone, four walls, a window, a floor, and a ceiling
+and serves as a starting model to implement the entire <code>DetailedHouse</code> model.
```

Comment:
I wouldn't say "entire" as `DetailedHouse5` serves as another starting point. This model is based on `DetailedHouse1`, but not exactly the same (as it uses two rectangular zone models instead of the separate building envelope models).